test/js/css-url-image-value.js

```diff
@@ -1,4 +1,13 @@
 suite('CSSURLImageValue', function() {
+  setup(function() {
+    this.element = document.createElement('div');
```

Comment:
Instead of creating a new element, can you just use `document.documentElement` instead?